sysutils/podman/Makefile

```diff
@@ -0,0 +1,31 @@
+PORTNAME=	podman
+DISTVERSION=	4.3.0.dev
+PORTVERSION=	4.3.0.dev
+CATEGORIES=	sysutils
+
+MAINTAINER=	dfr@FreeBSD.org
+COMMENT=	Manage Pods, Containers and Container Images
+WWW=		https://github.com/containers
+
+LICENSE=	GPLv2
+
+LIB_DEPENDS=	libgpgme.so:security/gpgme
+RUN_DEPENDS=	${LOCALBASE}/etc/containers/containers.conf.sample:sysutils/containers-common \
```

Comment:
This should probably also be changed to `containers-common>=0:sysutils/containers-common`.